src/lib/components/Widget.js

```diff
@@ -220,6 +222,7 @@ export const Widget = React.forwardRef((props, forwardedRef) => {
     try {
       setElement(vm.renderCode(vmInput) ?? "Execution failed");
     } catch (e) {
+      near.config.errorCallback({scope: ErrorScopes.Execution, message: e.message});
```

Comment:
We may want to report `src` here. Otherwise it would be difficult to figure out which widget throws this error